6.0/desktop/gui/theming.md

```diff
@@ -1,16 +1,15 @@
 # Theming
 
-Qooxdoo includes four themes:
+Qooxdoo includes five themes:
```

Comment:
Assuming that TangibleLight and TangibleDark are separate themes (I think they are), then there are six themes. To avoid future problems as themes are added and removed from this list, though, how about changing this to `Qooxdoo includes the following themes:`?